### PR TITLE
feat(maybe): add a function to convert maybe to an rxjs observable

### PR DESCRIPTION
**Pull request by williamareynolds**

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -8,14 +8,26 @@ export default [
       name: 'monads',
       file: `dist/${pkg.main}`,
       format: 'umd',
+      globals: {
+        rxjs: 'rxjs',
+        'rxjs/operators': 'rxjs/operators'
```
Comment (williamareynolds) [inline rollup.config.js]: Kind of wondering if this is a valid way to provide the global symbol for rxjs operators, or if it should be shortened to `operators` or something.

Comment (williamareynolds) [reply rollup.config.js]: Results in this in the UMD output:

```
(global = global || self, factory(global.monads = {}, global.rxjs, global['rxjs/operators']));
```

Comment (patrickmichalina) [reply rollup.config.js]: I don't know the answer to this one. I think the way you have it is good.

**Pull request merged**